app/components/subcases/subcase-item.js

```diff
@@ -89,6 +95,27 @@ export default class SubcaseItemSubcasesComponent extends Component {
     this.subcaseDocuments = sortPieces(pieces);
   }
 
+  @task
+  *loadSubcasePhases() {
+    this.phases = yield this.subcasesService.getSubcasePhases(this.args.subcase);
+  }
+
+  @task
+  *loadSubcaseIsApproved() {
+    const meeting = yield this.args.subcase.requestedForMeeting;
+    if (meeting?.isFinal) {
+      this.approved = !!(yield this.store.queryOne('agenda-item-treatment', {
+        'filter[subcase][id]': this.args.subcase.id,
+        'filter[decision-result-code][:id:]': [
+          CONSTANTS.DECISION_RESULT_CODE_IDS.GOEDGEKEURD,
+          CONSTANTS.DECISION_RESULT_CODE_IDS.KENNISNAME,
+        ].join(','),
+      }));
+    } else {
+      this.approved = false;
+    }
+  }
```

Comment:
This aims to replace the computed property `approved` on subcase.js, which i'm fine with.
We use it in more places, so eventually this could maybe move to a util instead, good for now atleast.

The computed `phases` on subcase was listening to the `approved` computed, which is now being called less so I found an instance where the phases was nog updated (refresh of page fixed it)
Have a subcase on a design agenda (only 1 or 2 phases, "ingediend" en "goedgekeurd")
in the same browser window:
- look at the subcase detail
- go to agenda, set formal ok and approve and close the agenda
- go back to subcase detail, the phases should now have "beslist op", but they don't, only after refresh.

Possible solution:
add `this.phases = yield this.subcasesService.getSubcasePhases(this.args.subcase);` to the afterModel hook of `cases/case/subcases/subcase/` and pass on that args to `SubcaseDescriptionView`, or load it that component I guess.
Then the `phases` computed should no longer be referenced.

Reply:
The changes in this PR shouldn't change the behaviour you described :thinking: 
The detail view of a subcase `cases.case.subcases.subcase.overview` is still using the computed `phases` & `approved` properties of the `subcase` model. So, unless I'm mistaken, in your example you're going form subcase overview to agenda back to subcase overview, so whatever is happening has nothing to do with the changes here (because those happen in `cases.case.subcases.overview`, and you're never passing that page.

Reply:
hmm, had to verify on dev branch and seems like you are right.
The problem is already present, my bad.

Could do the fix in a bug ticket since I think that fix will work, but ignore here